portfolio/src/main/java/com/google/sps/servlets/data/Comment.java

```diff
@@ -1,17 +1,24 @@
 package com.google.sps.data;
 
-/** An item on a todo list. */
+/** An item of comment entry. */
 public final class Comment {
 
-  private final long id;
-  private final String comment;
-  private final String user;
-  private final long timestamp;
+    private final long id;
+    private final String comment;
+    private final String user;
+    private final long timestamp;
 
-  public Comment(long id, String comment, String user, long timestamp) {
-    this.id = id;
-    this.comment = comment;
-    this.user = user;
-    this.timestamp = timestamp;
-  }
+    /**
+    * Thia is the constructor for Comment.
```

Comment:
typo: Thia.

And rephrase this comment the document into like either:

`Constructor for Comment.`

or 

` Creates a new <code>Comment</code> instance from id, content of the comment, timestamp`